src/Microsoft.OpenApi/Expressions/BodyExpression.cs

```diff
@@ -54,5 +54,16 @@ public override string Expression
                 return Body + PointerPrefix + Value;
             }
         }
+
+        /// <summary>
+        /// Gets the fragment string.
+        /// </summary>
+        public string Fragment
+        {
+            get
+            {
+                return Value;
+            }
+        }
```

Comment:
:smile:

Reply:
What does the laugh here mean? :)

Reply:
It's a smile. :) I like it that you are adding these properties. They match the runtime expression spec terminology.